src/SfxWeb/src/app/Models/DataModels/infrastructureJob.ts

```diff
@@ -0,0 +1,46 @@
+import { IRawInfrastructureJob, IRawInfraRepairTask, IRawRoleInstaceImpact } from '../RawDataTypes';
+import { TimeUtils } from 'src/app/Utils/TimeUtils';
+import { DataModelBase } from './Base';
+import { Observable, of } from 'rxjs';
+import { map } from 'rxjs/operators';
+import { StatusWarningLevel } from 'src/app/Common/Constants';
+import { IResponseMessageHandler } from 'src/app/Common/ResponseMessageHandlers';
+import { DataService } from 'src/app/services/data.service';
+
+export class InfrastructureJob extends DataModelBase<IRawInfrastructureJob> {
+    
+    IsActive : string;
```

Comment:
its generally avoided to redefine all the properties from the raw, instead either add getters to format one of the values and when needed reference .raw.<whatever value>